core/src/main/java/tachyon/web/WebInterfaceBrowseServlet.java

```diff
@@ -188,13 +188,22 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
       request.setAttribute("blockSizeByte", currentFileInfo.getBlockSizeBytes());
       if (!currentFileInfo.getIsDirectory()) {
         String tmpParam = request.getParameter("offset");
-        long offset = 0;
+        long relativeOffset = 0;
+        long offset;
         try {
           if (tmpParam != null) {
-            offset = Long.valueOf(tmpParam);
+            relativeOffset = Long.valueOf(tmpParam);
           }
         } catch (NumberFormatException nfe) {
-          offset = 0;
+          relativeOffset = 0;
+        }
+        tmpParam = request.getParameter("end");
```

Comment:
I know this isn't really related, but could tmpParam be changed to reasonable variable names?